playground/theme/recipes/menu.ts

```diff
@@ -0,0 +1,69 @@
+import { defineParts, defineRecipe } from '@pandacss/dev'
+import { menuAnatomy } from '@ark-ui/react'
+
+const parts = defineParts(menuAnatomy.build())
+
+const itemStyle = {
+  alignItems: 'center',
+  borderRadius: 'md',
+  cursor: 'pointer',
+  display: 'flex',
+  fontWeight: 'medium',
+  textStyle: 'sm',
+  transitionDuration: 'fast',
+  transitionProperty: 'background, color',
+  transitionTimingFunction: 'default',
+  h: '10',
+  px: '2.5',
+  mx: '1',
+  transition: 'background 0.2s ease',
+  _hover: {
+    bg: { base: 'gray.100', _dark: '#3A3A3AFF' },
+  },
+  _highlighted: {
+    bg: { base: 'gray.100', _dark: '#3A3A3AFF' },
+  },
+} as const
+
+export const menu = defineRecipe({
```

Comment:
wouldnt it make more sense to use defineSlotRecipes and ditch the defineParts ?

Reply:
@astahmer I'm not using slot recipes so I can apply the className on only the parent element.